pkgs/tools/system/nvtop/default.nix

```diff
@@ -1,18 +1,35 @@
-{ callPackage }:
+{ callPackage, stdenv }:
 let
   # this GPU families are supported "by-default" upstream (see https://github.com/Syllo/nvtop/blob/3a69c2d060298cd6f92cb09db944eded98be1c23/CMakeLists.txt#L81)
   # coincidentally, these families are also easy to build in nixpkgs at the moment
-  defaultGPUFamilies = [ "amd" "intel" "msm" "nvidia" "panfrost" "panthor" ];
+  defaultGPUFamilies = [
+    "amd"
+    "apple"
+    "intel"
+    "msm"
+    "nvidia"
+    "panfrost"
+    "panthor"
+  ];
   # these GPU families are partially supported upstream, they are also tricky to build in nixpkgs
   # volunteers with specific hardware needed to build and test these package variants
-  additionalGPUFamilies = [ "apple" "ascend" ];
-  defaultSupport = builtins.listToAttrs (builtins.map (gpu: { name = gpu; value = true; }) defaultGPUFamilies);
+  additionalGPUFamilies = [ "ascend" ];
+  defaultSupport = builtins.listToAttrs (
+    # apple can only build on darwin, and it can't build everything else, and vice versa
```

Comment:
I am a bit confused by this comment. It talks about building of a package, but the code below checks hostPlatform. Either comment should be rewritten, or the check should use stdenv.buildPlatform .

Reply:
And also I don't like how the ofborg builds are looking after this change. It seems that adding apple to defaultSupport list breaks nvtop-full build. Probably because of the hostPlatform check as well